internal/backend/mailbox_search.go

```diff
@@ -181,7 +181,7 @@ func (m *Mailbox) matchSearchKeyBcc(ctx context.Context, candidates []*snapMsg,
 			return false, err
 		}
 
-		return strings.Contains(value, key.GetText()), nil
+		return strings.Contains(strings.ToLower(value), strings.ToLower(key.GetText())), nil
```

Comment:
would it be worth create ?

 `func containsCaseIgnored(a,b string) bool {return strings.ToLower(a), strings.ToLower(b))`